docs/modules/usage/installation.mdx

```diff
@@ -87,6 +87,53 @@ If the required model does not exist in the list, you can toggle `Advanced` opti
 in the `Custom Model` text box.
 The `Advanced` options also allow you to specify a `Base URL` if required.
 
+### Getting an API Key
+
+OpenHands requires an API key to access language models. Here's how to get an API key from the recommended providers:
+
+#### Anthropic (Claude)
+
+1. **Create an Anthropic account**:
+   - Go to [console.anthropic.com](https://console.anthropic.com/) and sign up for an account if you don't already have one.
+
+2. **Generate an API key**:
+   - Once logged in, navigate to the [API Keys section](https://console.anthropic.com/settings/keys).
+   - Click "Create Key" and provide a name for your key.
+   - Select the appropriate key type (typically "Production" for regular use).
+   - Copy the generated API key immediately, as you won't be able to see it again.
+
+3. **Set up billing**:
+   - Go to [Billing settings](https://console.anthropic.com/settings/billing) to add a payment method.
+   - Consider setting usage limits to control costs.
+
```

Comment:
I wonder if we should just drop a link to anthropic docs instead of maintain a duplicate here? 

Don't have a strong preference either way